zipkin-server/src/main/java/zipkin/server/internal/ActuateCollectorMetrics.java

```diff
@@ -11,17 +13,11 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package zipkin.server.internal;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Date;
-import java.util.List;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.atomic.LongAdder;
-import java.util.function.Supplier;
-import org.springframework.boot.actuate.endpoint.PublicMetrics;
-import org.springframework.boot.actuate.metrics.Metric;
+import com.google.common.util.concurrent.AtomicDouble;
```

Comment:
Do we need to use `AtomicDouble`? Avoiding Guava would seem preferable.
Do we need strict atomicity for these gauges? Maybe a `Double` is sufficient.

Reply:
Thanks @shakuzen for the review. I think Gauge can observe values that are settable so I was using AtomicDouble. 
http://micrometer.io/docs/concepts#_gauges
Please correct If I am wrong.